src/cmd-buildupload

```diff
@@ -97,8 +99,9 @@ def s3_cp(args, src, dest, *s3_args):
     acl = f'--acl={args.acl}'
     dest = f's3://{args.url}/{dest}'
     print(f"Uploading: {dest}")
-    subprocess.check_call(['aws', 's3', 'cp', acl, src, dest, *s3_args],
-                          stdout=subprocess.DEVNULL)
+    if not args.dry_run:
```

Comment:
maybe print what it was going to do in the case of `--dry_run`?

Reply:
Fixed! :arrow_down: